recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "epicscorelibs" %}
-{% set version = "7.0.7.99.0.0" %}
```

Comment:
Source has no tags but upstream diff of commits identified by these versions indicates no packaging changes:
```
$ git diff cefd192d0cd4ab03e57cb908f14874f8ac56a1ae d470e21e369342eee570a587bfbdfaff4c6bb410
```

<details>

```diff
diff --git a/.github/workflows/build.yml b/.github/workflows/build.yml
index 8a7235d62..82b8d98b1 100644
--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -57,6 +57,11 @@ jobs:
             pyver: cp310-cp310
             piparch: manylinux2014_x86_64
 
+          - name: linux 3.11 amd64
+            os: ubuntu-latest
+            pyver: cp311-cp311
+            piparch: manylinux2014_x86_64
+
           # Linux py builds x64
           - name: linux 2.7 i686
             os: ubuntu-latest
@@ -133,6 +138,11 @@ jobs:
             python: "3.10"
             piparch: macosx_10_10_intel
 
+          - name: osx 3.11 intel
+            os: macos-latest
+            python: "3.11"
+            piparch: macosx_10_10_intel
+
           # Windows py builds
 
           ## missing Microsoft Visual C++ 9.0
@@ -170,6 +180,11 @@ jobs:
             python: "3.10"
             piparch: win_amd64
 
+          - name: win64 3.11
+            os: windows-latest
+            python: "3.11"
+            piparch: win_amd64
+
     steps:
       - uses: actions/checkout@v2
         with:
diff --git a/setup.py b/setup.py
index 3ef073559..6feda5381 100755
--- a/setup.py
+++ b/setup.py
@@ -8,12 +8,9 @@ from collections import defaultdict
 
 from setuptools import Command, Distribution
 from setuptools.command import install
-from distutils.command import build
 
 from setuptools_dso import Extension, setup, DSO, build_dso
 
-from distutils import log
-
 
 mydir = os.path.abspath(os.path.dirname(__file__))
 
@@ -26,7 +23,7 @@ from epicscorelibs.config import get_config_var
 # our choice of version suffix is constrained by PEP 440
 # so we always append .99.ABI.SRC to most recent upstream version
 # the following line is matched from cibuild.py
-package_version = '7.0.7.99.0.0'
+package_version = '7.0.7.99.0.1'
 
 assert package_version.split('.')[-3]=='99', package_version
 
@@ -76,7 +73,7 @@ class GenVersionError(Command):
                                   )
 
     def run(self):
-        log.info("In GetVersionError")
+        print("In GetVersionError")
         self.mkpath(self.build_temp)
         defs = {}
         readmake(defs, 'configure', 'CONFIG_BASE_VERSION')
@@ -84,9 +81,9 @@ class GenVersionError(Command):
 
         outfile = os.path.join(self.build_temp, 'epicsVersion.h')
         if self.dry_run:
-            log.info("Would write %s", outfile)
+            print("Would write %s", outfile)
         else:
-            log.info("Writing %s", outfile)
+            print("Writing %s", outfile)
             with open(outfile, 'w') as F:
                 F.write("""
 #ifndef INC_epicsVersion_H
@@ -119,9 +116,9 @@ class GenVersionError(Command):
 
         outfile = os.path.join(self.build_temp, 'generated.c')
         if self.dry_run:
-            log.info("Would write %s", outfile)
+            print("Would write %s", outfile)
         else:
-            log.info("Writing %s", outfile)
+            print("Writing %s", outfile)
             with open(outfile, 'w') as F:
                 F.write("""
 #include <stddef.h>
@@ -157,7 +154,7 @@ class GenVersionError(Command):
                                 defS.append(D)
 
                 if len(defM)==0 or len(defS)==0:
-                    log.warn("Warning: generated error string table is empty!")
+                    print("Warning: generated error string table is empty!")
                     # MSVC doesn't like zero length C arrays
                     # so give it something
                     defS.append({'macro':'S_generic', 'value':'1', 'desc':'placeholder'})
@@ -183,9 +180,9 @@ ERRSYMTAB_ID errSymTbl = &symTbl;
 
         outfile = os.path.join(self.build_temp, 'epicsVCS.h')
         if self.dry_run:
-            log.info("Would write %s", outfile)
+            print("Would write %s", outfile)
         else:
-            log.info("Writing %s", outfile)
+            print("Writing %s", outfile)
             with open(outfile, 'w') as F:
                 F.write("""
 #ifndef EPICS_VCS_VERSION
@@ -214,12 +211,12 @@ class Expand(Command):
                                   )
 
     def run(self):
-        log.info("In Expand")
+        print("In Expand")
         for template, outname, files in self.distribution.x_expand or []:
             template = os.path.normcase(template)
 
             outname = os.path.join(self.build_temp, os.path.normcase(outname))
-            log.info("expand %s -> %s", template, outname)
+            print("expand %s -> %s", template, outname)
 
             with open(template, 'r') as F:
                 tmpl = F.read()
@@ -257,7 +254,7 @@ class MakeAPIH(Command):
     def run(self):
         for stem, outname in self.distribution.x_api_h or []:
             outname = os.path.join(self.build_lib, 'epicscorelibs', 'include', os.path.basename(outname))
-            log.info('make %s -> %s', stem, outname)
+            print('make %s -> %s', stem, outname)
             out = '''
 #ifndef %(guard)s
 #define %(guard)s
@@ -315,7 +312,7 @@ class InstallHeaders(Command):
                                    ('build_temp', 'build_temp'),
                                   )
     def run(self):
-        log.info("In InstallHeaders")
+        print("In InstallHeaders")
         for pkg, files in (self.distribution.x_headers or {}).items():
             print("Install headers for", pkg)
             pkgdir = os.path.join(*[self.build_lib]+pkg.split('.'))
diff --git a/src/python/epicscorelibs/config.py b/src/python/epicscorelibs/config.py
index 6133709fc..3a24bba27 100644
--- a/src/python/epicscorelibs/config.py
+++ b/src/python/epicscorelibs/config.py
@@ -1,12 +1,11 @@
 """Host configuration information using EPICS names and build flags.
 
-Patterned after distutils.sysconfig
+Patterned after sysconfig
 """
 
 from copy import deepcopy
 import platform
 import sysconfig
-import distutils.sysconfig
 
 __all__ = (
     'get_config_var',
@@ -26,7 +25,7 @@ def _makeconf():
     }[platform.system()]
 
     # select epics CMPLR_CLASS
-    CC = sysconfig.get_config_var('CC') or distutils.sysconfig.get_config_var('CC')
+    CC = sysconfig.get_config_var('CC')
     if CC is None and osname=='WIN32':
         cmplrname = conf['CMPLR_CLASS'] = 'msvc'
     elif CC is None:

```
</details>